### ISSUE TITLE
Possibility to generate local.dat from saved username and password

### ISSUE THREAD

**Issue by Rainrainraind** [Healix/Gw2Launcher#185]
I was wondering if it's possible to created local.dat based on saved username and password? instead of entering manually one by one, as it's a lot of work to clone numerous accounts and key in the field one by one when you have lots of accounts

**Comment by Healix** [comment]
No. Use the automatic entry (under an account's launch options) if that's something you want. If you really want it saved in Local.dat, you have to login to save it. If you want to convert from automatic entry to Local.dat, disable automatic logins, launch (normal), select to remember the password, then login and exit.
